### PR TITLE
Supports custom schema parse depth

### PR DESCRIPTION
**Pull request by rbruggem**
Depending on the endpoint and resource template, we may need to parse the schema to different levels.

### CODE REVIEW

File: src/openapi-vuepress-markdown.ts [hunk 1]
```diff
@@ -5,6 +5,14 @@ import { Command } from 'commander'
 import { generateMarkdownFiles } from './markdownAdapters'
 const SwaggerParser = require('swagger-parser')
 
+function validateNumber(arg: string, name: string): boolean {
+    if (arg && isNaN(Number(arg))) {
```
Comment (jstasiak) [inline src/openapi-vuepress-markdown.ts]: Do we explicitly want to allow whitespace in the numbers and non-decimal numbers here?

**Review by jstasiak — COMMENTED** [review]
Two points here:

* A multi-letter single-dash options comment (like in here https://github.com/lune-climate/openapi-vuepress-markdown/pull/16)
* Is there an example of a situation where this feature is used?

**Comment by jstasiak** [comment]
Oh, I also believe the subject/title should use the imperative voice (so `Support ...`, not `... supports ...`)

File: src/openapi-vuepress-markdown.ts (hunk 1 above)
Comment (rbruggem) [reply src/openapi-vuepress-markdown.ts]: Not sure I understand.

Comment (jstasiak) [reply src/openapi-vuepress-markdown.ts]: What I mean is this is permitting inputs that we likely don't want permitted:


```
> isNaN(Number("0x123"))
false
> isNaN(Number("123 "))
false
> isNaN(Number("0o123"))
false
> isNaN(Number("0b111"))
false
```

Comment (rbruggem) [reply src/openapi-vuepress-markdown.ts]: Fixed @jstasiak

Comment (jstasiak) [reply src/openapi-vuepress-markdown.ts]: Cheers, sorry for being unclear

**Comment by jstasiak** [comment]
@rbruggem Almost everything in UNIX world uses single letters for short options (`-o`) and I think we should follow this convention too?

**Comment by rbruggem** [comment]
> @rbruggem Almost everything in UNIX world uses single letters for short options (`-o`) and I think we should follow this convention too?

You are right, thank you.

**Review by jstasiak — APPROVED** [review]
Ship

**Pull request merged**